src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -48,6 +49,7 @@ export const formBenefits = {
   [VA_FORM_IDS.FORM_20_0996]: 'Higher-level review',
   [VA_FORM_IDS.FORM_VA_2346A]: 'hearing aid batteries and accessories',
   [VA_FORM_IDS.FORM_5655]: 'financial status report',
+  [VA_FORM_IDS.FORM_22_1990_MEB]: 'education benefits',
```

Comment:
please move this up with the other 1990 forms

Reply:
Done, thx.